app/components/index-buttons.hbs

```diff
@@ -0,0 +1,24 @@
+<div class="heading">
+  {{#if (eq @status "active")}}
+    <h2 data-test-status>I am doing a task</h2>
+  {{/if}}
+  {{#if (eq @status "idle")}}
+    <h2 data-test-status>I am idle</h2>
+  {{/if}} 
+  {{#if (eq @status "ooo")}}
+    <h2 data-test-status>I am OOO</h2>
+  {{/if}} 
+</div>
+<div class="buttons">
+  {{#if (eq @status "active")}}
+    <button class="idle" type="button" {{on "click" (fn @onUpdateStatus 'idle')}}><span data-test-button-text>Change status to ‘Idle’</span></button>
+    <button class="small" type="button" {{on "click" (fn @onUpdateStatus 'ooo')}}><span>Mark yourself as OOO</span></button>
+  {{/if}}   
+  {{#if (eq @status "idle")}}
+    <button class="active" type="button" data-test-button {{on "click" (fn @onUpdateStatus 'active')}}><span data-test-button-text>Change status to ‘Doing a task’</span></button>
+    <button class="small" type="button" {{on "click" (fn @onUpdateStatus 'ooo')}}><span>Mark yourself as OOO</span></button>
+  {{/if}}
+  {{#if (eq @status "ooo")}}
+    <button class="small" type="button" {{on "click" (fn @onUpdateStatus 'active')}}><span>Mark yourself as Active again</span></button>
+  {{/if}}
+</div>

```

Comment:
We are using BEM for CSS to maintain consistency with class names
You can read up on what's the convention, check `tasks` files if you have doubts (read up first and then see the code, we should get used to learning from docs)

Reply:
Okay